README.md

```diff
@@ -21,8 +21,8 @@ If you need help using or installing Node.js, please use the
 
 ### Official Resources
 
-* [Website][]
-* [Contributing to the project][]
+* [nodejs.org](https://nodejs.org/)
+* Contributing: [CONTRIBUTING.md](./CONTRIBUTING.md)
```

Comment:
This change would leave two dead links at the bottom of the page. I'd prefer if you could just update the links on the bottom instead. The relative link to CONTRIBUTING.md could be problematic, how about just adding a link to the file on master instead?